.rubocop_todo.yml

```diff
@@ -17,7 +17,7 @@ Metrics/AbcSize:
 # Offense count: 220
 # Configuration parameters: AllowURI, URISchemes.
 Metrics/LineLength:
-  Max: 135
+  Max: 138
```

Comment:
We'll treat is as tech debt for the time being. Just get rubocop in and passing so nothing gets worse, then do spikes to fix certain things at 1 time (eg reduce all line lengths to something shorter)